core/src/main/java/org/apache/seata/core/rpc/netty/AbstractNettyRemotingServer.java

```diff
@@ -68,28 +69,28 @@ public AbstractNettyRemotingServer(ThreadPoolExecutor messageExecutor, NettyServ
 
     @Override
     public Object sendSyncRequest(String resourceId, String clientId, Object msg, boolean tryOtherApp)
-        throws TimeoutException {
+            throws TimeoutException, IOException {
         Channel channel = ChannelManager.getChannel(resourceId, clientId, tryOtherApp);
         if (channel == null) {
-            throw new RuntimeException("rm client is not connected. dbkey:" + resourceId + ",clientId:" + clientId);
+            throw new IOException("rm client is not connected. dbkey:" + resourceId + ",clientId:" + clientId);
```

Comment:
Why is it considered an IO exception when a channel does not exist, rather than a runtime exception?

Reply:
Then I believe it would be more appropriate to wrap it as a SeataException.